source/app-development/tutorial-ps-to-quota.rst

```diff
@@ -0,0 +1,451 @@
+      <tr>
+        <th>Filesystem</th>
+        <th>Blocks</th>
+        <th>Blocks Quota</th>
+        <th>Blocks Limit</th>
+        <th>Blocks Grace</th>
+        <th>Files</th>
+        <th>Files Quota</th>
+        <th>Files Limit</th>
+        <th>Files Grace</th>
+      </tr>
+      <% @quotas.each do |quota| %>
+      <tr>
+        <td><%= quota.filesystem %></td>
+        <td><%= quota.blocks %></td>
+        <td><%= quota.blocks_quota %></td>
+        <td><%= quota.blocks_limit %></td>
+        <td><%= quota.blocks_grace %></td>
+        <td><%= quota.files %></td>
+        <td><%= quota.files_quota %></td>
+        <td><%= quota.files_limit %></td>
+        <td><%= quota.files_grace %></td>
+      </tr>
+      <% end %>
+    </table>
+
+These changes should not require an app restart. Go to the launched app and reload the page to see the changes.
+
+Brand App
+---------
+
+The app is looking good, but the details page still shows the app title "Passenger App Processes". To change this and the icon, edit the manifest.yml:
+
+.. code:: diff
+
+    -name: Passenger App Processes
+    -description: Display your running Passenger app proceseses in a table
+    +name: Quota
+    +description: Display quotas
+    +icon: fa://hdd-o
+
+* The icon follows format of ``fa://{FONTAWESOMENAME}`` where you replace ``{FONTAWESOMENAME}`` with an icon from http://fontawesome.io/icons/.
+  In this case we are using ``fa-hdd-o`` which we write in the manifest as ``fa://hdd-o``: http://fontawesome.io/icon/hdd-o/
+
+Publish App
+-----------
+
+Publishing an app requires two steps:
+
+#. Updating the manifest.yml to specify the category and optionally subcategory, which indicates where in the dashboard menu the app appears.
+#. Having an administrator checkout a copy of the production version to a directory under /var/www/ood/apps/sys
+
+
+Steps:
+
+#. Add category to manifest so app appears in Files menu:
+
+    .. code:: diff
+
+        name: Quota
+        description: Display quotas
+        icon: fa://hdd-o
+        +category: Files
+        +subcategory: Utilities
+
+#. Version these changes. Click Shell button on app details view, and then commit the changes:
+
+    .. code:: sh
+
+       git add .
+       git commit -m "update manifest for production"
+
+       # if there is an external remote associated with this, push to that
+       git push origin master
+
+#. As the admin, sudo copy or clone this repo to production
+
+    .. code:: sh
+
+       # as sudo on OnDemand host:
+       cd /var/www/ood/apps/sys
+       git clone /users/PZS0562/efranz/ondemand/dev/quota
+
+
+#. Reload the dashboard.
+
+.. figure:: /images/app-dev-tutorial-ps-to-quota-published.png
+   :align: center
+
+   Every user can now launch the Quota from the Files menu.
+
+.. warning::
+
+   Accessing this new app for the first time will cause your NGINX server to restart, killing all websocket connections, which means resetting your active Shell sessions.
```

Comment:
"which means any active web-based Shell sessions will reset!"

Personal preference make it active and add mention that this only applies to web-based shells.